benchmark/templates/fio_direct_read_test.cfg.j2

```diff
@@ -3,10 +3,11 @@ randrepeat=0
 runtime=180
 
 [fio_direct_read_test]
-filename={{ bench_direct_device_name }}
-direct=1
 rw=read
-ioengine=libaio
```

Comment:
If we do this are we not removing some benchmarks in favour of others?

We should maybe just make the ioengine option template-able, similar to iodepth or numjobs then specify the additional checks inside the group_vars/benchmark_hosts/00-default.yml

Reply:
I can do that.  My current plan is to make a single fio.cfg.j2 that converts the yml definition of an fio config to ini